website/docs/r/sfn_state_machine.html.markdown

```diff
@@ -35,6 +36,65 @@ EOF
 }
 ```
 
+### Basic (Express Workflow)
+
+```hcl
+# ...
+
+resource "aws_sfn_state_machine" "sfn_state_machine" {
+  name     = "my-state-machine"
+  role_arn = "${aws_iam_role.iam_for_sfn.arn}"
+  type     = "EXPRESS"
+
+  definition = <<EOF
+{
+  "Comment": "A Hello World example of the Amazon States Language using an AWS Lambda Function",
+  "StartAt": "HelloWorld",
+  "States": {
+    "HelloWorld": {
+      "Type": "Task",
+      "Resource": "${aws_lambda_function.lambda.arn}",
+      "End": true
+    }
+  }
+}
+EOF
+}
+```
+
+### Logging
+
+~> *NOTE:* See the [AWS Step Functions Developer Guide](https://docs.aws.amazon.com/step-functions/latest/dg/welcome.html) for more information about enabling Step Function logging.
+
+```hcl
+# ...
+
+resource "aws_sfn_state_machine" "sfn_state_machine" {
+  name     = "my-state-machine"
+  role_arn = "${aws_iam_role.iam_for_sfn.arn}"
+
+  definition = <<EOF
+{
+  "Comment": "A Hello World example of the Amazon States Language using an AWS Lambda Function",
+  "StartAt": "HelloWorld",
+  "States": {
+    "HelloWorld": {
+      "Type": "Task",
+      "Resource": "${aws_lambda_function.lambda.arn}",
+      "End": true
+    }
+  }
+}
+EOF
+ 
+  logging_configuration {
+     log_destination = "${aws_cloudwatch_log_group.log_group_for_sfn.arn}"
+     include_execution_data = true
+     level = "ERROR"
```

Comment:
These need to be aligned for the linting to be successful and the automated check to succeed

Reply:
hopefully this fixes it.

Reply:
My comment was towards the equal signs being aligned in HCL

Reply:
I thought it didn't like the equal signs being misaligned but it looks like that fixed it!